expression/builtin_time.go

```diff
@@ -1447,6 +1449,84 @@ func (b *builtinUnixTimestampSig) eval(row []types.Datum) (d types.Datum, err er
 	return
 }
 
+type timestampFunctionClass struct {
+	baseFunctionClass
+}
+
+func (c *timestampFunctionClass) getFunction(args []Expression, ctx context.Context) (builtinFunc, error) {
+	return &builtinTimestampSig{newBaseBuiltinFunc(args, ctx)}, errors.Trace(c.verifyArgs(args))
+}
+
+type builtinTimestampSig struct {
+	baseBuiltinFunc
+}
+
+func (b *builtinTimestampSig) eval(row []types.Datum) (types.Datum, error) {
+	args, err := b.evalArgs(row)
+	if err != nil {
+		return types.Datum{}, errors.Trace(err)
+	}
+	return builtinTimestamp(args, b.ctx)
+}
+
+// https://dev.mysql.com/doc/refman/5.5/en/date-and-time-functions.html#function_timestamp
+func builtinTimestamp(args []types.Datum, ctx context.Context) (d types.Datum, err error) {
```

Comment:
You can merge  builtinTimestamp() with builtinTimestampSig.eval() to one function.